op-chain-ops/README.md

```diff
@@ -34,6 +34,7 @@ cmd/
 ├── check-derivation              - Check that transactions can be confirmed and safety can be consolidated
 ├── check-ecotone                 - Checks for Ecotone network upgrade
 ├── check-fjord                   - Checks for Fjord network upgrade
+├── check-prestate                - Checks a fault proof absolute prestate's chain compatibility
```

Comment:
Is it worth having a little more documentation, e.g. how to invoke the command?

Reply:
Added an example of executing it. Really we could do with cleaning up these check tasks to be subcommands of a single executable (noted in the vision part of the README) but that's a bigger change than I have time for at the moment.